buildspecs/linux_x86-64_cmprssptrs_valhalla_nestmates.spec

```diff
@@ -0,0 +1,276 @@
+		<property name="javatestPlatform" value="linux_x86-64_cr"/>
+		<property name="jclMaxSecurityPolicyDefine" value=" &quot;-Djava.security.policy=http://jcl1.ottawa.ibm.com/testres/java.policy&quot;"/>
+		<property name="jclMemoryMax" value="-Xmx64m"/>
+		<property name="jclOSStackSizeMax" value=""/>
+		<property name="jgrinderTestingSupported" value="true"/>
+		<property name="jitTestingOptLevel" value="optlevel=warm"/>
+		<property name="localRootPath" value="$(J9_UNIX_ROOT)"/>
+		<property name="longLimitCmd" value=""/>
+		<property name="main_shortname" value="xa64_valhalla_nestmates"/>
+		<property name="os.lineDelimiter" value="unix"/>
+		<property name="platform_arch" value="amd64"/>
+		<property name="svn_stream" value=""/>
+		<property name="uma_make_cmd_ar" value="ar"/>
+		<property name="uma_make_cmd_as" value="as"/>
+		<property name="uma_make_cmd_cc" value="cc"/>
+		<property name="uma_make_cmd_cpp" value="cpp -E -P"/>
+		<property name="uma_make_cmd_cxx" value="c++"/>
+		<property name="uma_make_cmd_cxx_dll_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_cxx_exe_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_dll_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_exe_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_interp_gcc" value="gcc-4.6"/>
+		<property name="uma_make_cmd_ranlib" value="ranlib"/>
+		<property name="uma_processor" value="amd64"/>
+		<property name="uma_type" value="unix,linux"/>
+		<property name="uma_use_cflags_on_asflag_line" value="true"/>
+	</properties>
+	<features>
+		<feature id="cmprssptrs"/>
+		<feature id="combogc"/>
+		<feature id="core"/>
+		<feature id="dbgext"/>
+		<feature id="gpu"/>
+		<feature id="se"/>
+		<feature id="se60_26"/>
+		<feature id="se7"/>
+		<feature id="se70_27"/>
+	</features>
+	<source>
+		<project id="com.ibm.jvmti.tests"/>
+		<project id="compiler"/>
+	</source>
+	<flags>
+		<flag id="arch_x86" value="true"/>
+		<flag id="build_SE6_package" value="true"/>
+		<flag id="build_autobuild" value="true"/>
+		<flag id="build_dropToHursley" value="true"/>
+		<flag id="build_dropToToronto" value="true"/>
+		<flag id="build_j2se" value="false"/>
+		<flag id="build_java8" value="false"/>
+		<flag id="build_java9" value="false"/>
+		<flag id="build_openj9" value="false"/>
+		<flag id="build_openj9JDK8" value="false"/>
+		<flag id="build_product" value="false"/>
+		<flag id="build_vmContinuous" value="true"/>
+		<flag id="env_hasFPU" value="true"/>
+		<flag id="env_littleEndian" value="true"/>
+		<flag id="gc_batchClearTLH" value="true"/>
+		<flag id="gc_debugAsserts" value="true"/>
+		<flag id="gc_idleHeapManager" value="true"/>
+		<flag id="gc_inlinedAllocFields" value="true"/>
+		<flag id="gc_minimumObjectSize" value="true"/>
+		<flag id="gc_tlhPrefetchFTA" value="true"/>
+		<flag id="graph_cmdLineTester" value="true"/>
+		<flag id="graph_compile" value="true"/>
+		<flag id="graph_enableModularityTesting" value="true"/>
+		<flag id="graph_enableTesting" value="true"/>
```

Comment:
The panama spec has enableTesting and enableTesting_Java8 set to false, although not sure if it matters with all the build parameters set to false.

Reply:
The flag part was set based off the current `linux_ppc-64_cmprssptrs_le_valhalla_nestmates.spec`, not sure if the `enableTesting` flag actually matter in OpenJ9 builds as this will not be run under Java8,9 (ie Axxon)
I updated `enableTesting_Java8` to false to make it look more consistent